config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -45,6 +45,13 @@ periodics:
       - name: service
         mountPath: /etc/service-account
         readOnly: true
+      resources:
+        requests:
+          cpu: 4
```

Comment:
we should set the number of workers flag

Reply:
Why not let it pick it up automatically?

Reply:
Is it correctly detecting it?
There are various mechanisms for finding this info on linux, and IIRC not all of them are correctly masked by containers.

Reply:
I can add some logging so we can be sure